hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCleanConfig.java

```diff
@@ -118,28 +120,32 @@ public class HoodieCleanConfig extends HoodieConfig {
           + "the minimum number of file slices to retain in each file group, during cleaning.");
 
   public static final ConfigProperty<String> CLEAN_TRIGGER_STRATEGY = ConfigProperty
-      .key("hoodie.clean.trigger.strategy")
+      .key("hoodie.cleaner.trigger.strategy")
       .defaultValue(CleaningTriggerStrategy.NUM_COMMITS.name())
+      .withAlternatives("hoodie.clean.trigger.strategy")
       .markAdvanced()
       .withDocumentation(CleaningTriggerStrategy.class);
 
   public static final ConfigProperty<String> CLEAN_MAX_COMMITS = ConfigProperty
-      .key("hoodie.clean.max.commits")
+      .key("hoodie.cleaner.trigger.max.commits")
       .defaultValue("1")
+      .withAlternatives("hoodie.clean.max.commits")
       .markAdvanced()
       .withDocumentation("Number of commits after the last clean operation, before scheduling of a new clean is attempted.");
 
   public static final ConfigProperty<String> CLEANER_INCREMENTAL_MODE_ENABLE = ConfigProperty
-      .key("hoodie.cleaner.incremental.mode")
+      .key("hoodie.cleaner.incremental.enabled")
       .defaultValue("true")
+      .withAlternatives("hoodie.cleaner.incremental.mode")
       .markAdvanced()
       .withDocumentation("When enabled, the plans for each cleaner service run is computed incrementally off the events "
           + "in the timeline, since the last cleaner run. This is much more efficient than obtaining listings for the full "
           + "table for each planning (even with a metadata table).");
 
   public static final ConfigProperty<String> FAILED_WRITES_CLEANER_POLICY = ConfigProperty
-      .key("hoodie.cleaner.policy.failed.writes")
+      .key("hoodie.cleaner.failed.writes.policy")
```

Comment:
Do we prefer `clean` or `cleaner` or `cleaning`? Maybe just stays with `clean`.

Reply:
It makes sense, changed all `cleaner` to `clean`.